.github/workflows/hosted_runners.yml

```diff
@@ -1334,13 +1336,12 @@ jobs:
         if %errorlevel% neq 0 exit /b %errorlevel%
 
     - name: Locate the packages
-      working-directory: ${{ steps.build_paths.outputs.PACKAGE_BUILD }}
       id: packages
-      shell: bash
+      shell: powershell
```

Comment:
This doesn't look like powershell syntax to me

Reply:
Well, `echo` and `ls` definitely work under a powershell shell, they are just aliases (the `.FullName` part is the giveaway for powershell). Mostly this was to get the path to the files in the Windows format.

If you prefer I can change them to their non-aliased version; it was mostly for the reduced verbosity to me.

Reply:
It was the `$()` I wasn't sure of